src/Modal.js

```diff
@@ -177,4 +35,131 @@ const styles = StyleSheet.create({
   },
 });
 
+const Modal = ({
+  backdropStyle,
+  children,
+  contentStyle,
+  isVisible,
+  onBackdropPress,
+  onHide,
+  ...otherProps
+}) => {
+  const animVal = new Animated.Value(0);
+  const { state, setState } = useState({
+    isVisible: isVisible,
+    isMounted: false,
+  });
+  const { height, width } = useWindowDimensions();
+  const backdropAnimatedStyle = {
+    opacity: animVal.interpolate({
+      inputRange: [0, 1],
+      outputRange: [0, MODAL_BACKDROP_OPACITY],
+    }),
+  };
+  const contentAnimatedStyle = {
+    transform: [
+      {
+        translateY: animVal.interpolate({
+          inputRange: [0, 1],
+          outputRange: [height, 0],
+          extrapolate: "clamp",
+        }),
+      },
+    ],
+  };
+
+  useEffect(() => {
+    setState({
+      ...state,
+      isMounted: true,
+    });
+    if (state.isVisible) {
+      setState({ isVisible: true });
```

Comment:
Hm, I'm missing something here. If the state is already visibile, why are why setting it again to true? 🤔